### PR TITLE
More decriptive descriptions for PRED1-4 mutations

### PR DESCRIPTION
**Pull request by RadaRadaRada**
The old descriptions for pred1-4 were flavorful but gave almost no indication what exactly they did. I wanted them more descriptive for a long time, but because I am not a native speaker and my grammar is not the best, I did not dare to try and kept discarding this idea. But, the best way to get something, is to do it yourself, so here is my attempt at this.

### CODE REVIEW

**Comment by chaosvolt** [comment]
Ah, useful. I'm not sure about the somewhat awkward second half of Hunter's description, but the rest seems good.

File: data/json/mutations.json [hunk 1]
```diff
@@ -2520,7 +2519,7 @@
         "id" : "PRED4",
         "name" : "Apex Predator",
         "points" : 2,
-        "description" : "It's good to be on top of the food chain.",
+        "description" : "Your mind and brain have adapted to your new violent existence, resulting in almost effortless mastering of combat skills. Unfortunately, your critical thinking, so characteristic to humans, has atrophied: -3 INT.",
```
Comment (Rivet-the-Zombie) [inline data/json/mutations.json]: Please double space after sentence breaks.

File: data/json/mutations.json [hunk 2]
```diff
@@ -2479,7 +2479,7 @@
         "id" : "PRED1",
         "name" : "Culler",
         "points" : 2,
-        "description" : "You've had a revelation: by killing the weaker creatures, who would only die anyway, you preserve resources for those better able to survive.",
+        "description" : "By killing the weaker creatures, who would only die anyway, you preserve resources for those better able to survive.  You are less bothered by death of others: their weakness invited these fates upon them.",
```
Comment (kevingranade) [inline data/json/mutations.json]: The "You've had a revelation:" phrase served an important function here of making it clear that this is a belief that the survivor has, not fact.

File: data/json/mutations.json [hunk 3]
```diff
@@ -2503,7 +2502,7 @@
         "id" : "PRED3",
         "name" : "Predator",
         "points" : 3,
-        "description" : "Let's not overthink things.  You get hungry, you find prey, you kill it and you eat it.",
+        "description" : "You consider yourself something other than human and no longer empathise with them.  Combat skills are easy to learn and maintain, but your critical thinking, so characteristic to these creatures, suffers: -1 INT.",
```
Comment (illi-kun) [inline data/json/mutations.json]: I'm not sure it's okay to show such explicit stat changes in the description (i.e. ```INT -1```), it's better to describe it verbally.

Comment (Rivet-the-Zombie) [inline data/json/mutations.json]: >it's better to describe it verbally.

Agreed.

Comment (RadaRadaRada) [inline data/json/mutations.json]: But all other mutations which modify stats explicitly tell the player exact stat changes.

Comment (Rivet-the-Zombie) [inline data/json/mutations.json]: Okay I just checked, and you're right, I was wrong.

This is fine.

Comment (illi-kun) [inline data/json/mutations.json]: Yeah, it's fine if it's in line with other mutations.

Comment (kevingranade) [inline data/json/mutations.json]: I disagree, I'd rather keep it all descriptive and move the other ones to
match.

Comment (chaosvolt) [inline data/json/mutations.json]: So long as the descriptive format adequately hints at what the trait actually does, which seems to be the objective of the PR.

**Comment by RadaRadaRada** [comment]
Removed explicit stat change descriptions from PREDx. Should I do it with other mutations too?

**Comment by kevingranade** [comment]
Making it clear what the traits achieve is fine, but several of these are entirely rewritten, discarding the tone of the original in favor of just stating what the trait does.
Can you just add to or adjust the descriptions to state what they do instead of replacing them?

**Comment by chaosvolt** [comment]
Let me see if I can suggest appropriate descriptions...PRED1 looks acceptable to me.

PRED2: "You're getting quite used to hunting and killing ...game.  For food. You find it easier to hone your hunting skills."

PRED3: "Let's not overthink things.  You get hungry, you find prey, you kill it and you eat it. Your ability to hunt has been honed further, but your intellect suffers for it."

PRED4: "It's good to be on top of the food chain. You feel that hunting and killing comes naturally to you, but your critical thinking has atrophied further."

These are still a tad derpy and change the tone a bit, but they do at least retain their original flavor text.

**Pull request merged**